<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
<<<<<<< HEAD
    <width>1004</width>
=======
    <width>839</width>
>>>>>>> e1a18cf145c2204eaae565638cb0d50a38580484
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
<<<<<<< HEAD
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>971</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Overwiew</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>841</width>
        <height>451</height>
       </rect>
      </property>
      <property name="title">
       <string>Status</string>
      </property>
      <widget class="QLabel" name="temp_png">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>150</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Desktop/WDS/temperatura.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="hum_png">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>260</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Desktop/WDS/wilgotnosc.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lux_png">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>370</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Desktop/WDS/naswietlenie.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTextEdit" name="main_text">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>30</y>
         <width>341</width>
         <height>101</height>
        </rect>
       </property>
      </widget>
      <widget class="QLCDNumber" name="tempLCD">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>150</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QLCDNumber" name="humLCD">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>260</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QLCDNumber" name="luxLCD">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>360</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>331</width>
         <height>251</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="connectButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="device_edit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="getdata">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>20</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Data</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exit_button">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>20</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Chartsxx</string>
     </attribute>
     <widget class="QCustomPlot" name="plot" native="true">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>851</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionShow_charts">
   <property name="text">
    <string>Show charts</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
=======
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Overwiew</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="connectButton">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="test_button">
          <property name="text">
           <string>Test</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="exit_button">
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Overwiew</string>
          </property>
          <widget class="QComboBox" name="comboBox">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>221</width>
             <height>41</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="temp_png">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>480</x>
             <y>120</y>
             <width>50</width>
             <height>50</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../Desktop/WDS/temperatura.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>480</x>
             <y>230</y>
             <width>50</width>
             <height>50</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../Desktop/WDS/wilgotnosc.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>480</x>
             <y>340</y>
             <width>50</width>
             <height>50</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../Desktop/WDS/naswietlenie.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QTextEdit" name="main_text">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>130</y>
             <width>431</width>
             <height>281</height>
            </rect>
           </property>
          </widget>
          <widget class="QLCDNumber" name="lcdNumber">
           <property name="geometry">
            <rect>
             <x>570</x>
             <y>130</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLCDNumber" name="lcdNumber_2">
           <property name="geometry">
            <rect>
             <x>570</x>
             <y>230</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLCDNumber" name="lcdNumber_3">
           <property name="geometry">
            <rect>
             <x>570</x>
             <y>340</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Charts</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect">
   <property name="icon">
    <iconset>
     <normaloff>img/connect.png</normaloff>img/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
 </widget>
>>>>>>> e1a18cf145c2204eaae565638cb0d50a38580484
 <resources/>
 <connections/>
</ui>
